codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -71,6 +71,7 @@ class FileCodesignVisitor {
     'CODESIGN_TEAM_ID': '',
     'APP_SPECIFIC_PASSWORD': ''
   };
+  Set<String> whitelist = {'FlutterEmbedder.framework.zip', 'FlutterMacOS.dSYM.zip'};
```

Comment:
I would prefer to have something closer to the artifact creation. Doing this way will mean that we need to update this tool every time a new artifact is added.

Reply:
Maybe just logging that a file was not code signed because it didn't have code sign metadata?

Reply:
@christopherfujino what do you think?

Reply:
Ah i see. Good point! yeah need to change BUILD.gn to add another file

Reply:
Yeah, I think the build rules is the right place to add this.